Apps/WebClient/src/ClientApp/app/models/ConfigData.ts

```diff
@@ -0,0 +1,64 @@
+/** A collection of configuration items for use by Health Gateway and approved applications. */
```

Comment:
Should we have all of these as nullable types?  I would think that we should confirm the existence of our configuration when we load it and then the rest of the app doesn't have to add checks.